src/commands/uplink/NewEvent.ts

```diff
@@ -0,0 +1,158 @@
+import Command from '../../Command.js';
+import CommandBinaryBuffer from '../../CommandBinaryBuffer.js';
+import * as Events from '../../constants/events.js';
+
+
+const COMMAND_ID = 0x15;
+const COMMAND_TITLE = 'NEW_EVENT';
+// connect/disconnect events are biggest
+const COMMAND_BODY_MAX_SIZE = 6;
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface EventBase {}
+
+type EventTime = EventBase & {
```

Comment:
interface?

Reply:
type, interface - as basic, type for values

Reply:
types are for simple types, interfaces - for objects

Reply:
fixed